setup.py

```diff
@@ -295,6 +295,9 @@ def run(self):
     'ordereddict; python_version < "2.7"',
     'enum34; python_version < "3.4"'
 ]
+# Python 3.10 workaround as enum34 not available
+if sys.version_info >= (3, 10):
+    requirements.remove('enum34; python_version < "3.4"')
```

Comment:
This shouldn't be needed. The intent of `enum34; python_version < "3.4"` is that enum34 only gets installed for python versions lower than 3.4.

Reply:
I hit a bug without this and how python=3.10 gets resolved vs python 3.4 IIRC

Reply:
Oof. I dug and found that this bug was fixed 2 years ago: https://github.com/pypa/pip/pull/7166. What version of pip are you using?

Reply:
I wasn't doing anything, it was whatever CI here is using :)